config/routes.rb

```diff
@@ -1,4 +1,13 @@
 Rails.application.routes.draw do
+  root 'home#index'
+  # devise_for :users, :controllers => {:registrations => "my_devise/registrations"}
+  devise_for :users, controllers: { sessions: 'devise/sessions' }
+
+  
+  resources :languages do
+    delete 'remove'
+    post 'add'
+  end
   devise_for :users
   root 'welcome#homepage'
```

Comment:
Now that there is a `root` at the beginning of this file, this line should be deleted.